tools/ramdump/HowToUseRamdump.md

```diff
@@ -206,3 +206,28 @@ Call Trace of Crashed Task :[appmain] with pid :2 and state :TSTATE_TASK_RUNNING
 [<40c9fec>] task_start+0x64         [Line 173 of \"task/task_start.c\"]
 ********************************************************************
 ```
+## How to port RAMDUMP
+To port ramdump for a new board, do the following steps:
+
+1. Add low level functions to receive and transfer characters through UART like: up_putc, up_puts, up_getc, etc.
```

Comment:
Because this is a porting guide, we should list all of necessary functions. Let's remove `etc` and list all of them.
Additionally, we had better add the description of each function.